Example/GalleryDemo/GalleryDemo/Sources/ViewController.swift

```diff
@@ -3,6 +3,7 @@ import Gallery
 import Lightbox
 import AVFoundation
 import AVKit
+import SVProgressHUD
```

Comment:
Do we really need to add a dependency just to show a loading spinner? Can't you just use `UIActivityIndicatorView`?

Reply:
It's in the demo, and SVProgressHUD works quickly in this case, and it's in ObjC 😬